src/context.js

```diff
@@ -1,26 +1,51 @@
 // @flow
 import React, { type Node } from 'react';
+import LocalStorage from './storages/local';
 
 // $FlowFixMe
 const { Provider, Consumer } = React.createContext({ data: {}, setResponse: () => {} });
 
 type ProviderProps = {
-  children: Node
+  children: Node,
+  storagePrefix: string
 };
 type ProviderState = {
   data: Object
 };
+type ResponsePair = {
+  response: any,
+  error: any
+};
+
+const DefaultStoragePrefix = 'react_loads_';
```

Comment:
PascalCase constants? Our types are PascalCased. Lets make this SHOUTING_CASE.

Reply:
I_HATE_SHOUTING_CASE_SO_MUCH!